lms/djangoapps/instructor/views/api.py

```diff
@@ -1264,6 +1264,8 @@ def get_students_features(request, course_id, csv=False):  # pylint: disable=red
         'level_of_education': _('Level of Education'),
         'mailing_address': _('Mailing Address'),
         'goals': _('Goals'),
+        'city': _('City'),
+        'country': _('Country'),
     }
 
     if is_course_cohorted(course.id):
```

Comment:
Thanks @cahrens. Please tag me on any changes so I can reflect updates in the doc.